web/modules/custom/usa_admin_styles/benefitsCategoryLanguageFilter.js

```diff
@@ -0,0 +1,64 @@
+/*
+Terms in the "Benefits Category" taxonomy can be English or Español, and
+the "Benefits Category Search" field displayed while editing a Basic Page
+content type shows all the items by default.
+
+This script filters those categories by the content's selected language.
+
+This script assumes that each category includes its language at the end of the <label> element
+*/
+
+(function (jQuery, Drupal, drupalSettings) {
+
+    storeLanguageInDataAttribute(jQuery);
+
+    filterCategoriesByLanguage(jQuery);
+
+    jQuery('#edit-langcode-0-value').change(function(){
```

Comment:
the `change` shorthand is deprecated along with a bunch of (maybe all?) other events - <https://api.jquery.com/change-shorthand/>

For forward compatibility, they recommend doing `.on('change', function(){`

Reply:
Good catch